cs/v1/CS_v1.0.0.yaml

```diff
@@ -528,297 +562,229 @@ components:
-          example: 2103N
-        carrierImportVoyageNumber:
-          type: string
-          x-stoplight:
-            id: e9np9hrh7c934
-          maxLength: 50
-          example: 2103S
-          description: The identifier of an import voyage. The carrier-specific identifier of the import Voyage.
-        carrierExportVoyageNumber:
-          type: string
-          x-stoplight:
-            id: 5u650xz8q6zfm
-          example: 2103N
-          maxLength: 50
-          description: The identifier of an export voyage. The carrier-specific identifier of the export Voyage.
-        departure:
-          $ref: '#/components/schemas/Place'
-        arrival:
-          $ref: '#/components/schemas/Place'
     PortSchedule:
       title: Port Schedule
       type: object
-      x-stoplight:
-        id: mgxbp9alny3q0
       required:
         - location
       properties:
         location:
-          x-stoplight:
-            id: ksjcrliv4kobm
-          oneOf:
-            - $ref: '#/components/schemas/UNLocationLocation'
-            - $ref: '#/components/schemas/FacilitySMDGLocation'
-          discriminator:
-            propertyName: locationType
-            mapping:
-              FACS: '#/components/schemas/FacilitySMDGLocation'
-              UNLO: '#/components/schemas/UNLocationLocation'
+          $ref: '#/components/schemas/PortScheduleLocation'
         vesselSchedules:
           type: array
-          x-stoplight:
-            id: oqtnha66e947j
-          minItems: 1
           items:
             $ref: '#/components/schemas/Schedule'
     Schedule:
       title: Schedule
-      x-stoplight:
-        id: czyyep580i801
       type: object
       required:
-        - carrierServiceName
-        - carrierServiceCode
         - isDummyVessel
-        - carrierImportVoyageNumber
-        - carrierExportVoyageNumber
         - timestamps
+        - serviceCarriers
       properties:
-        carrierServiceName:
-          type: string
-          x-stoplight:
-            id: g8e6j84n6mcw8
-          description: The name of the service.
-          maxLength: 50
-          example: Great Lion Service
-        carrierServiceCode:
-          type: string
-          x-stoplight:
-            id: ys8adwf0o370o
-          description: The carrier-specific code of the service for which the schedule details are published.
-          maxLength: 11
-          example: FE1
         universalServiceReference:
           type: string
-          x-stoplight:
-            id: 18655doinhur2
           description: 'A global unique service reference, as per DCSA standard, agreed by VSA partners for the service. The service reference must match the regular expression pattern: SR\d{5}[A-Z]. The letters SR followed by 5 digits, followed by a checksum-character as a capital letter from A to Z.'
           example: SR12345A
           pattern: ^SR\d{5}[A-Z]$
           maxLength: 8
+          minLength: 8
+        servicePartners:
```

Comment:
This name is not aligned with the required fields. Either rename this field to `serviceCarriers` or change the required field to `servicePartners`